dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyAPI.kt

```diff
@@ -56,12 +57,19 @@ interface CompanyAPI {
         produces = ["application/json"]
     )
     /**
-     * A method to retrieve specific companies identified by their company names
-     * If an empty string is passed as company name, all companies in the data store will be returned.
-     * @param companyName identifier used to search for companies in the data store (can also be an empty string)
+     * A method to retrieve specific companies identified by their company names identifier or stock index
+     * If an empty string is passed as search argument, all companies in the data store will be returned.
+     * If selectedIndex is not null, all companies with the given stock index will be returned.
+     * @param searchString string used for substring matching
+     * @param selectedIndex StockIndex Enum used to filter against stock indices
```

Comment:
Grundsätzliche Frage bzw. ein Gedanke; habe ich vllt wegen Krankheitsausfall verpasst:

Ist es problematisch bzw. ein Mangel an Funktion, dass man nicht auf mehrere Stock Indizes filtern kann?
Wird man das später vllt noch brauchen?

Reply:
Aktuell brauchen wir die Indizies nur für die klickbare Liste im Frontend. Dort kann immer nur ein Index ausgewählt werden. Es kann sein das wir in der Zukunft auch nach mehreren Indizes filtern wollen aber dann kann man ja die Funktionalität bauen wenn man sie braucht.